conformance/chaos/actor.go

```diff
@@ -31,10 +29,16 @@ type Actor struct{}
 type CallerValidationBranch int64
 
 const (
+	// CallerValidationBranchNone causes no caller validation to take place.
 	CallerValidationBranchNone CallerValidationBranch = iota
+	// CallerValidationBranchTwice causes Runtime.ValidateImmediateCallerAcceptAny to be called twice.
 	CallerValidationBranchTwice
-	CallerValidationBranchAddrNilSet
-	CallerValidationBranchTypeNilSet
+	// CallerValidationBranchIs causes caller validation against CallerValidationArgs.Addrs.
+	CallerValidationBranchIs
+	// CallerValidationBranchIsReceiver causes validation that the caller was also the receiver.
+	CallerValidationBranchIsReceiver
```

Comment:
```suggestion
	CallerValidationBranchIsSelf
```

I think it would be more deterministic if we used `CallerValidationBranchIs` with the explicit address we expect? We do seem to always know it ahead of time.